frontend/package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "frontend",
-  "version": "1.1.0-24.12",
```

Comment:
I see a package lock change but no package change, intended?

Reply:
yep, now it matches the version on package.json